google-cloud-testing/storage-benchwrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# storage-benchwrapper
+
+storage-benchwrapper is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```
+mvn clean install -DskipTests=true (one time)
+cd google-cloud-testing/storage-benchwrapper
+export STORAGE_EMULATOR_HOST=localhost:8080
+mvn clean install exec:java -DskipTests=true -Dport=8081
+```
+
+## Generating .proto sources
+
+Sources are generated automatically as part of the protobuf-maven-plugin plugin,
+but if you'd like to generate them yourself to see the output you can run:
+
+```
+cd google-cloud-testing/storage-benchwrapper
+protoc \
+    --plugin=protoc-gen-grpc-java=build/exe/java_plugin/protoc-gen-grpc-java \
+    --java_out=src/main/java \
+    --grpc-java_out=src/main/java \
```

Comment:
If you change this to `target/generated-sources/protobuf/java/` it will use the same location the maven plugin does and will remove the possible confusion of there being duplicates that would need to be manually cleaned up.

Alternatively, if you want to provide the command that will run just the protobuf generation in maven you could use: `mvn generate-sources`

Reply:
I included this here because vscode doesn't seem to know how to interpret the target/ directory well enough to understand import paths of imported things. So, these commands are included if you want to wack away at some temp things that live in src/main (which vscode will happily understand).

Reply:
Ah, fair enough.

Do we want to add a `.gitignore` to this module with the paths of the generated files under `src/main/java` set to ignore so they aren't accidentally committed? If not, the resulting build failure might be sufficient. Up to you.

Reply:
Build failure is OK for me. (less magic) Thank you!